web/src/i18n/translations/ja.yml

```diff
@@ -1,3 +1,33 @@
+Download: ''
```

Comment:
@nourbalaha , ask for translation before merging.

Reply:
I will ask Baba-san to translate once you and Kyle approve this PR.